addons/purchase_stock/tests/test_replenish_wizard.py

```diff
@@ -4,7 +4,8 @@
 
 from odoo.addons.stock.tests.common import TestStockCommon
 
-from odoo import fields
+from odoo import fields, Command
```

Comment:
```suggestion
from odoo import Command, fields
```
Nitpick: Imports should be in [alphabetical order](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports).